src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php

```diff
@@ -22,108 +23,161 @@ class AblyBroadcaster extends Broadcaster
      */
     protected $ably;
 
+    /**
+     * Used for setting expiry of issues tokens.
+     *
+     * @var int|mixed
+     * @default 1 hr
+     */
+    private $tokenExpiry = 3600;
+
+    /**
+     * Default channel capabilities, all public channels are by default given subscribe, history and channel-metadata access
+     * Set as per https://ably.com/docs/core-features/authentication#capability-operations.
+     *
+     * @var array|\string[][]
+     */
+    private $defaultChannelClaims = [
+        'public:*' => ['subscribe', 'history', 'channel-metadata'],
+    ];
+
     /**
      * Create a new broadcaster instance.
      *
-     * @param  \Ably\AblyRest  $ably
+     * @param $ably \Ably\AblyRest
```

Comment:
```suggestion
     * @param  \Ably\AblyRest  $ably
     * @param  array $config
```
First variable type and then the variable name
The second parameter is missing (probably an array)

Reply:
Fixed!